api/src/test/java/org/openmrs/test/matchers/HasGlobalErrors.java

```diff
@@ -57,7 +57,7 @@ protected boolean matchesSafely(Errors item) {
 		if (code == null) {
 			return item.hasGlobalErrors();
 		} else {
-			return item.getGlobalErrors().stream().filter(c -> code.equals(c.getCode())).findFirst().isPresent();
```

Comment:
Did you look into? "findFirst().isPresent() doesn't fit the behavior of anyMatch() for null elements since findFirst() forbids finding null elements while anyMatch() allows matching them"

Reply:
I see. So, should I revert to anyMatch(), right?

Reply:
Did you first of all confirm before agreeing with me? 😊

Reply:
Yes, I've checked that findFirst() forbids null, and anyMatch() does not, so I agree with you